docx/text/paragraph.py

```diff
@@ -427,8 +427,9 @@ def set_li_lvl(self, styles, prev, ilvl):
     @property
     @cache
     def run_text(self):
-        if self.runs:
-            return ''.join(r.text for r in self.runs)
+        runs = self.runs
+        if runs:
+            return ''.join(r.text for r in runs)
```

Comment:
I assume you made this change for performance reasons?

Reply:
That is correct.